engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -2178,6 +2178,15 @@ public void execute(String[] args, ConsoleReader reader) throws Exception
 
   }
 
+  private ApplicationReport findApplicationReportFromAppNameOrId(String appNameOrId)
+  {
+    ApplicationReport app = getApplication(appNameOrId);
+    if (app == null) {
+      app = getApplicationByName(appNameOrId);
+    }
+    return app;
```

Comment:
All the callers throw the identical exception when the returned app is null. Can we just throw that exception here and the caller can always assume app is non-null?